locales/en.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    HomeHeaderRight: {
```

Comment:
So you want to abandon nesting? As it is now done with Issues title, it will be like this - HomeHeaderRight_title?